ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Flex } from '@patternfly/react-core';
+
+import { SelectOption } from '@patternfly/react-core/deprecated';
```

Comment:
Do you think it would be worthwhile to migrate to the new PatternFly Select component for these instead of the deprecated component?

I know the functionality in the new one doesn't quite cover the number of use cases the old one does, so up to you if you think the effort is worth it now given the release schedule.

Reply:
Yes, good point. We could migrate to the new PF Select component. I think given the release schedule, we could try to make a one-off case for this specific case for now. I can add a TODO to refactor and make it more reusable.